user-guides/voice-and-video-conferencing/README.md

```diff
@@ -0,0 +1,19 @@
+# Voice and Video Conferencing
+
+In Rocket.Chat you have the ability to start video and audio conferences.
+
+This is done by using [Jitsi Meet](https://jitsi.org/jitsi-meet/) through Rocket.Chat.
+
+> Note: For this feature to be available for normal users, the server admin must activate it before. Please see [Setting up video conferencing](../../administrator-guides/setting-up-video-conferencing)
+
+## Starting a Video Conference
+
+To start a video conference you can simply open the channel actions menu (the three dots on the top right) and select `Video Chat`.
+
+> If you cant see the `Video Chat` option on the channel actions menu, please contact your system admin, since video conferencing can be allowed only on private channels or on all channels depending on the server configuration.
+
+Other users will be able to join via `Click to Join!` button that is sent as a message on the channel that the conference is being held.
+
+## Screen Sharing
+
+You can start screen sharing by selecting `Share screen` in the jitsi widget.
```

Comment:
May want to note that with Firefox, this just works.  But with Chrome, they must add this plugin https://chrome.google.com/webstore/detail/rocketchat-screen-share/nocfbnnmjnndkbipkabodnheejiegccf    [please verify before adding ].